src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -1112,13 +1111,14 @@ namespace AppInstaller::CLI::Workflow
                 directives.Insert(s_Directive_Module, PropertyValue::CreateString(moduleNameWide));
 
                 winrt::hstring description;
+                auto localized = std::wstring{ Resource::String::ConfigureExportUnitDescription };

```

Comment:
If you are going to support localization here, you need to use the format specifier and the localization formatting functionality.  You can't just append strings together.

Also, you didn't do it for the `Install` above.